SimPEG/EM/Base.py

```diff
@@ -183,7 +188,12 @@ def MfRhoIDeriv(self,u):
         """
             Derivative of :code:`MfRhoI` with respect to the model.
         """
-        return self.mesh.getFaceInnerProductDeriv(self.curModel.rho, invMat=True)(u) * self.curModel.rhoDeriv
+
+        dMfRhoI_dI = -self.MfRhoI**2
+        dMf_drho = self.mesh.getFaceInnerProductDeriv(self.curModel.rho)(u)
+        return dMfRhoI_dI * ( dMf_drho * (-Utils.sdiag(self.curModel.rho**2) * self.curModel.sigmaDeriv) )
```

Comment:
@rowanc1 : this is b/c of a bug in the propmap derivs (using rhoDeriv as per line 196 will fail)